source/daplink/cmsis-dap/DAP_vendor.c

```diff
@@ -101,6 +101,9 @@ uint32_t DAP_ProcessVendorCommand(const uint8_t *request, uint8_t *response) {
         // uart read
         int32_t read_len = 62;
         read_len = uart_read_data(response + 1, read_len);
+        if (read_len) {
+            main_blink_cdc_led(MAIN_LED_FLASH);
+        }
```

Comment:
The changes in this file were introduced as a fix for https://github.com/ARMmbed/DAPLink/issues/714